docs/src/getting-started/basics.md

```diff
@@ -241,19 +241,19 @@ For a more concrete example, see the `test_contract_calling_contract` function i
 
 ## Connecting to existing contracts
 
-If you already have a deployed contract and want to call its methods using the SDK,  but without deploying it again, all you need is the contract ID of your deployed contract. You can skip the whole deployment setup and call `::new(contract_id, provider, wallet)` directly. For example:
+If you already have a deployed contract and want to call its methods using the SDK,  but without deploying it again, all you need is the contract ID of your deployed contract. You can skip the whole deployment setup and call `::new(contract_id, wallet)` directly. For example:
```

Comment:
Just realised that this example isn't accurate. If you have an already deplyed contract, then you already have a node. So you *have* to use a provider that connects to said node to connect the wallet to it. The instruction in this version would confuse users.

Reply:
This was the case even before my changes. A workflow that involves connecting a provider to a node, instead of starting it via `launch_provider_and_get_wallet` is not introduced. I fear confusion might go the other way as well.

Reply:
Can you please open a `docs` issue for this part then. This would be very confusing to a new user I think.